CMakeLists.txt

```diff
@@ -44,15 +63,16 @@ else()
     target_link_libraries(${PROJECT_NAME} tensorflow-lite)
 endif()
 
-
 if(CLFML_FACE_DETECTOR_ENABLE_CORAL_TPU)
-target_link_libraries(${PROJECT_NAME} edgetpu)
-set(FACE_DETECTOR_CORAL_MODEL_PATH ${CMAKE_CURRENT_LIST_DIR}/models/Coral/face_detection.tflite)
-target_compile_definitions(${PROJECT_NAME} PUBLIC -DFACE_DETECTOR_ENABLE_CORAL_SUPPORT=1 
-                                                  -DCFML_FACE_DETECTOR_CORAL_MODEL_PATH="${FACE_DETECTOR_CORAL_MODEL_PATH}")
-
+    target_link_libraries(${PROJECT_NAME} edgetpu)
+    set(FACE_DETECTOR_CORAL_MODEL_PATH ${CMAKE_CURRENT_LIST_DIR}/models/Coral/face_detection.tflite)
+    target_compile_definitions(${PROJECT_NAME} PUBLIC -DFACE_DETECTOR_ENABLE_CORAL_SUPPORT=1 
+                                                      -DCFML_FACE_DETECTOR_CORAL_MODEL_PATH="${FACE_DETECTOR_CORAL_MODEL_PATH}")
 endif()
 
+if (CLFML_ROS2_PACKAGE_BUILD)
+    target_link_libraries(${PROJECT_NAME} ${cv_bridge_LIBRARIES})
```

Comment:
I see that the only dependency of the library is cv_bridge, it might be better to merge the first IF with all the findpackages with this if for the cv_bridge library.

like this:

```cmake
if (CLFML_ROS2_PACKAGE_BUILD)
   find_package(cv_bridge REQUIRED)
   target_link_libraries(${PROJECT_NAME} ${cv_bridge_LIBRARIES})
endif()
```

Reply:
By removing this if-statement you don't force users of the library on dependencies they might or might not use. As the other find_package statements are mainly used for the example code you provided

Reply:
ok!